src/client/graphics/layers/RadialMenu.ts

```diff
@@ -266,8 +267,26 @@ export class RadialMenu implements Layer {
       .style("pointer-events", "none");
   }
 
-  tick() {
-    // Update logic if needed
+  async tick() {
+    // Only update when menu is visible
+    if (!this.isVisible || this.clickedCell === null) return;
+    const myPlayer = this.g.myPlayer();
+    if (myPlayer === undefined || !myPlayer.isAlive()) return;
```

Comment:
`find()` returns `undefined`, but `myPlayer()` returns `null`.
```suggestion
    if (myPlayer === null || !myPlayer.isAlive()) return;
```